drivers/at/at.c

```diff
@@ -295,8 +336,6 @@ ssize_t at_send_cmd_get_lines(at_dev_t *dev, const char *command,
         goto out;
     }
 
-    /* only clear the response buffer after sending the command,
```

Comment:
Removed because it implies the command and response buffer may overlap, which by function signature would be UB.